contracts/src/StakeTable.sol

```diff
@@ -8,10 +8,13 @@ import { LightClient } from "../src/LightClient.sol";
 import { EdOnBN254 } from "./libraries/EdOnBn254.sol";
 import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
 
+// TODO: remove this when contract is made upgradable
```

Comment:
remove this?

Reply:
https://github.com/EspressoSystems/espresso-sequencer/pull/2647/commits/c9b2501c2ddf0366f7c2b140b20f0c41a4586513